app/models/user.rb

```diff
@@ -1,3 +1,4 @@
 class User < ApplicationRecord
   include Clearance::User
+  validates :name, presence: true
```

Comment:
Can we get a test for `tries with an invalid name` to go with the ones for invalid email/username?